### DISCUSSION TITLE
Developer/Plugin category

### DISCUSSION THREAD

**Discussion by Cooper-JTS** [Budibase/budibase#14786] [Ideas]
This isn’t necessarily an idea for the actual Budibase platform but more for the community. I think it would be very helpful to have a category for developers/plugins so that we can communicate and discuss more on that side of the platform than just the general platform itself.

Along those same lines, I’ve had a hard time finding any real documentation of the regarding the development side of Budibase. 

Examples: 
What object do I reference to get data from a table?
What Budibase specific objects and functions are available to build a component?
How does Budibase refer to the data in the actions drawer and how do I get data to it.
Etc…

**Comment by joebudi** [comment]
Added
